client/src/components/layout/IndexPage.js

```diff
@@ -0,0 +1,172 @@
+import React, { useState } from "react"
+import fetchPlayer from "../fetchRequests/fetchPlayer.js"
+import fetchStats from "../fetchRequests/fetchStats.js"
+import FormError from "./FormError.js"
+import statsList from "../../constants/statsList.js"
+import PlayerTile from "./PlayerTile.js"
+import StatTile from "./StatTile.js"
+
+const IndexPage = (props) => {
+  const [player, setPlayer] = useState({ name: "", season: "" })
+  const [selectedPlayers, setSelectedPlayers] = useState([])
+  const [selectedStats, setSelectedStats] = useState([])
+  const [errors, setErrors] = useState({})
+
+  const statsOptions = [""].concat(statsList).map((stat) => {
+    return (
+      <option key={stat} value={stat}>
+        {stat}
+      </option>
+    )
+  })
+
+  const handlePlayerInputChange = (event) => {
+    setPlayer({
+      ...player,
+      [event.currentTarget.name]: event.currentTarget.value,
+    })
+  }
+
+  const handleStatsInputChange = (event) => {
+    let newStats = selectedStats.concat(event.currentTarget.value)
+    setSelectedStats(newStats)
+  }
+
+  const handleClearTable = (event) => {
+    event.preventDefault()
+    setSelectedPlayers([])
+    setSelectedStats([])
+  }
+
+  const validateInput = (payload) => {
+    setErrors({})
+    const { name, season } = payload
+    let newErrors = {}
+    if (season.trim() === "") {
+      newErrors = {
+        ...newErrors,
+        season: "is required",
+      }
+    }
+
+    if (name.trim() === "") {
+      newErrors = {
+        ...newErrors,
+        name: "is required",
+      }
+    }
+
+    setErrors(newErrors)
+    return newErrors
+  }
+
+  const handlePlayerSubmit = async (event) => {
+    event.preventDefault()
+    const validationErrors = validateInput(player)
+    if (Object.keys(validationErrors).length === 0) {
+      let fetchedPlayerData = await fetchPlayer(player.name)
+      const fetchedStatsData = await fetchStats(
+        fetchedPlayerData.id,
+        player.season
+      )
```

Comment:
if you wanted to be super pro, line 67 - 71 could be moved into their own function, that you then import into this API

no need to do that right now though, nice job